integration-tests/tests/taiko_circuit_input_builder.rs

```diff
@@ -0,0 +1,26 @@
+#![cfg(feature = "circuit_input_builder")]
+
+use integration_tests::{build_circuit_input_builder_block, log_init};
+
+macro_rules! declare_tests {
+    ($test_name:ident, $block_num:expr) => {
+        #[tokio::test]
+        async fn $test_name() {
+            log_init();
+            build_circuit_input_builder_block($block_num).await;
+        }
+    };
+}
+
+// This test builds the complete circuit inputs for the block that has
+// only one anchor
+declare_tests!(test_circuit_input_builder_block_anchor_only, 137947);
```

Comment:
`137947 `, `137932`
All these constant block numbers can be set to `const`, so they can be shared with other place.

Reply:
Updated to use const